idc_index/__init__.py

```diff
@@ -9,3 +9,5 @@
 from ._version import version as __version__
 
 __all__ = ["__version__"]
+
+from .index import IDCClient
```

Comment:
@fedorov I added this import such that we can import `IDCClient` directly, instead of importing `index` and then accessing it via `index.IDCClient`. I see this got removed, is this because we have any rule that prevents adding such imports on the package level or did it just got lost in the process? I prefer this way as it simplifies importing a lot and enhances ux in my opinion but I'm open for counter-arguments ;)

Some of the errors are related to the now missing import, so I wait for your reply before fixing them.

Thx for looking into this!!

Reply:
The linter doesn't like the unused import... I push a workaround, let me know what you think ;)

Reply:
Indeed, I just accepted the corrections that we proposed by the linter/ruff. Looks good to me!